### ISSUE TITLE
[REVIEW] Internal Review, test, and scan all pages for "ALTOONA, PA" PART 1 System and facility pages in Devshop [VAMC]

### ISSUE THREAD

**Issue by SgardnerPMP** [department-of-veterans-affairs/va.gov-team#7063]
## User Story or Problem Statement
As a Veteran Web user, I must have full accessibility a compliance to my web applications and tools, so I can use the VAMC system pages in any GUI, UE, including mobile and reader environments.

## Goal
_Perform initial page testing, review, initial Accessibility scan with AXE-Coconut _

## Objectives or Key Results this is meant to further
- _Increase overall quality experience for veteran's accessibility_
- _Assist and reduce time for complete accessibility and 508 compliance checks_

## Resources - Tools - Documentation
[Chrome AXE-Coconut](https://chrome.google.com/webstore/detail/axe-coconut-web-accessibi/iobddmbdndbbbfjopjdgadphaoihpojp/related?hl=en)
[Chrome AXE](https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd/related?hl=en-US)

## Tasks
_Complete review of system and facility pages, first manual review and testing of links, Axe, Axe Coconut, and Lighthouse on the following test lines from above Test resource:
- [ ] Test lines 1 through 11
- [ ] Test lines 32 through 46
* All lines highlighted in Green

## Acceptance Criteria
* All pages must have been documented in Spreadsheet
* AXE., AXE-Coconut issues should be documented, Lighthouse testing must be screenshot and loaded into Spreadsheet for Accessibility and 508 teams.   Any items on page that appear incorrect need to be documented or screenshot and documented.  Leave comments in cells if additional information is required.
